src/js/sticky-nav.js

```diff
@@ -1,12 +1,13 @@
-export function initStickyNav() {
+function stickToTop() {
   const nav = document.querySelector('.main-header');
```

Comment:
This `const` is used only once and there is no benefit of having the `const` assignment. You can just move the `document.querySelector` to the end of the function.